sechub-schedule/src/main/java/com/mercedesbenz/sechub/domain/schedule/SchedulerJobBatchTriggerService.java

```diff
@@ -90,9 +90,10 @@ protected void postConstruct() {
     }
 
     // default 10 seconds delay and 5 seconds initial
-    @MustBeDocumented("Job scheduling is triggered by a cron job operation - default is 10 seconds to delay after last execution. " + "The initial delay is "
-            + DEFAULT_INITIAL_DELAY_MILLIS + " milliseconds is defined. It can be configured different,so when you need to startup a cluster "
-            + "time shifted, simply change the initial delay values in your wanted way.")
+    @MustBeDocumented(value = "Job scheduling is triggered by a cron job operation - default is 10 seconds to delay after last execution. "
+            + "The initial delay is " + DEFAULT_INITIAL_DELAY_MILLIS
+            + " milliseconds is defined. It can be configured different,so when you need to startup a cluster "
```

Comment:
The second part is difficult to understand. How about:

It can be configured differently. This is useful when you need to startup a cluster. Simply change the initial delay values in to allow the cluster to startup.

Reply:
You are right about the last sentence.
- I also fixed here the grammar issue like done before for `AutoCleanupConstants#TRIGGER_STEP_MUST_BE_DOCUMENTED` .
- I also fixed the last sentence in `AutoCleanupConstants#TRIGGER_STEP_MUST_BE_DOCUMENTED` .